scripts/ansible

```diff
@@ -64,16 +64,17 @@ elif [ -f /etc/centos-release ]; then
 #elif [ -f /etc/debian_version ] || (grep -qi raspbian /etc/*elease) ; then
 #elif [ ! -f /etc/centos-release ] && [ ! -f /etc/fedora-release ] && [ ! -f /etc/olpc-release ]; then
 elif [ -f /etc/debian_version ]; then    # Includes Debian, Ubuntu & Raspbian
-
-    echo -e "\napt update; install dirmngr; PPA to /etc/apt/sources.list.d/iiab-ansible.list\n"
-    apt update
-    apt -y install dirmngr    # Raspbian needs.  Formerly: python-pip python-setuptools python-wheel patch
-    echo "deb http://ppa.launchpad.net/ansible/ansible/ubuntu bionic main" \
+    IS_U20=`cat /etc/os-release | grep UBUNTU_CODENAME= | awk -F "=" '{print $2}'`
+    if [ "$IS_20" != "focal" ]; then
+        echo -e "\napt update; install dirmngr; PPA to /etc/apt/sources.list.d/iiab-ansible.list\n"
+        apt update
+        apt -y install dirmngr
+        echo "deb http://ppa.launchpad.net/ansible/ansible/ubuntu bionic main" \
          > /etc/apt/sources.list.d/iiab-ansible.list
 
-    echo -e '\nIF YOU FACE ERROR "signatures couldn'"'"'t be verified because the public key is not available" THEN REPEATEDLY RE-RUN "sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 93C4A3FD7BB9C367"\n'
-    apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 93C4A3FD7BB9C367
-
+        echo -e '\nIF YOU FACE ERROR "signatures couldn'"'"'t be verified because the public key is not available" THEN REPEATEDLY RE-RUN "sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 93C4A3FD7BB9C367"\n'
+        apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 93C4A3FD7BB9C367
+    fi
```

Comment:
Can ` apt install ansible` be considered on Ubuntu 20.04, perhaps with an else clause here?

Reply:
Why? The PPA and required key are not being used to install, ansible is being provide by Ubuntu, just setting up the PPA and key are skipped when 'focal' is matched. What do you propose?

Reply:
I was assuming @georgejhunt manually ran something like `apt install ansible` on Ubuntu 20.04 pre-releases, and that scripts/Ansible here would emulate his approach here?

(But perhaps I have the details wrong...and there's a better way?)

Reply:
Think that might be the case and perhaps left out the ansible dependencies, only to install them later in roles/...